# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../silo/be-crosscompilable.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./second/Makefile.orig	2004-06-28 23:34:28.348996456 +0200
+++ ./second/Makefile	2004-06-28 23:34:36.316785168 +0200
@@ -85,7 +85,7 @@
 	$(CC) $(CFLAGS) -c -o $@ $<
 
 util:	util.c
-	$(CC) -DSMALL_RELOC=$(SMALL_RELOC) -DLARGE_RELOC=$(LARGE_RELOC) -o $@ $<
+	$(HOST_CC) -DSMALL_RELOC=$(SMALL_RELOC) -DLARGE_RELOC=$(LARGE_RELOC) -o $@ $<
 
 clean:
 	$(RM) *.o fs/*.o second* silotftp* util fs/libfs.a
--- ./common/Makefile.orig	2004-06-28 23:32:28.987142216 +0200
+++ ./common/Makefile	2004-06-28 23:33:46.917295032 +0200
@@ -12,12 +12,15 @@
 
 OBJS = sdiv.o rem.o udiv.o urem.o jmp.o printf.o console.o prom.o tree.o stringops2.o\
 	stringops1.o ffs.o divdi3.o udivdi3.o
-PROGRAMS = bin2h
 
-all: $(OBJS) $(PROGRAMS)
+bin2h:
+	$(HOST_CC) $(CFLAGS) -o bin2h bin2h.c
+
+all: $(OBJS) bin2h
 
 prom.o: prom.c
 	$(CC) $(CFLAGS) -c -Wa,-Av9 -o prom.o prom.c
 
 clean:
 	$(RM) $(PROGRAMS) *.o
+
